source/test/F0.Minesweeper.Components.Tests/Pages/Game/Modules/CellTests.razor

```diff
@@ -118,25 +121,6 @@
 		componentUnderTest.Find("button").Click();
 	}
 
-	[Fact]
-	public void OnRightClick_StatusManagerReturnsFlagged_ChangesTextToFlagged()
-	{
-		// Arrange
-		RenderFragment expectedMarkup = @<div><button class="f0-cell f0-cell-covered">⚐</button></div>;
-
-		cellStatusManagerMock.Setup((manager) => manager.CanMoveNext(CellInteractionType.RightClick, null)).Returns(true);
-		cellStatusManagerMock.Setup((manager) => manager.MoveNext(CellInteractionType.RightClick, null)).Returns(CellStatusType.Flagged);
-
-		ComponentParameter parameter = ComponentParameterFactory.Parameter(nameof(Cell.Location), new Location(1, 1));
-		IRenderedComponent<Cell> componentUnderTest = RenderComponent<Cell>(parameter);
-
-		// Act
-		componentUnderTest.Find("button").ContextMenu();
-
-		// Assert
-		componentUnderTest.MarkupMatches(expectedMarkup);
-	}
-
 	[Fact]
 	public void OnRightClick_StatusManagerReturnsUnsure_ChangesTextToQuestionMark()
```

Comment:
We removed similar tests to this - now I am unsure if this one is still valid and expressive. Is it?

Reply:
Yes, it is still valid. It is the singular necessary unit test to have a right click on a Cell covered. 

Maybe the new (albeit more clunky) name `OnRightClick_StatusManagerReturnsNewStatus_ChangesTextToCorrectVisualisationFromVisualisationManager` makes this more clear.